libcxx/include/__memory_resource/monotonic_buffer_resource.h

```diff
@@ -38,7 +38,6 @@ class _LIBCPP_AVAILABILITY_PMR _LIBCPP_EXPORTED_FROM_ABI monotonic_buffer_resour
     _LIBCPP_HIDE_FROM_ABI size_t __allocation_size() {
       return (reinterpret_cast<char*>(this) - __start_) + sizeof(*this);
     }
-    void* __try_allocate_from_chunk(size_t, size_t);
```

Comment:
Oops, both `__try_allocate_from_chunk` functions are ABI-critical, which is probably indicated by lack of `_LIBCPP_HIDE_FROM_ABI`. They must still exist, although I'm not sure whether it's possibly to change their behaivor (if so, they can just return `nullptr`).

Reply:
Those functions are on the ABI boundary, we can't remove them or change their behavior in a backwards incompatible way because pre-existing programs compiled against older versions of the headers will contain calls to this external function.

Unless nothing in the headers was ever calling into this function, in which case we might be able to remove it from the dylib. Is that the situation?

Edit: Doing a bit of searching, it seems like nobody's depending on that symbol, which would support that it was actually never needed as part of the ABI. I think it would be OK to remove it. You will need to update the `libcxx/lib/abi/*.abilist` files and `libcxx/lib/abi/CHANGELOG.TXT`.

Reply:
thanks, i will change